docs/course_materials/exercises/05_Control_structures/homework_while.md

```diff
@@ -6,21 +6,21 @@
 
 Create a file *while_average.py*
 
-Write a Python program that gets integer numbers from the user in a loop (use 'input'),
-The loop should break when the user enters the number 0.
-At the end, the program will print the sum and the average of the numbers.
+Write a Python program that gets integer numbers from the user in a loop (use 'input'),<br>
+The loop should break when the user enters the number 0.<br>
+At the end, the program should print the sum and the average of the numbers.
 
 for example: 
 
 the user input is : 15, 16, 12, 0
 The output should be: 
-Sum of 15 + 16  + 12 = 43 
-Average of the numbers = 14.3 
+Sum = 43 
+Average = 14.3 
 
 ### positive_negative.py
 
 Create a file *positive_negative.py*
 
 Write a program to enter the numbers till the user wants (The loop should break when the user enters the number 0.)
```

Comment:
(-) Write a program to enter the numbers till the user wants (The loop should break when the user enters the number 0.)
(+) Write a Python program that gets integer numbers from the user in a loop (use 'input'),< br >
(+) The loop should break when the user enters the number 0.< br >

Reply:
done.